controlplane/rosa/api/v1beta2/rosacontrolplane_types.go

```diff
@@ -99,6 +99,26 @@ type RosaControlPlaneSpec struct { //nolint: maligned
 	// Autoscaling specifies auto scaling behaviour for the MachinePools.
 	// +optional
 	Autoscaling *expinfrav1.RosaMachinePoolAutoScaling `json:"autoscaling,omitempty"`
+
+	// +kubebuilder:validation:Optional
```

Comment:
Nit: These `// +kubebuilder:` comment tags should be underneath the field not above, we've used this pattern in most if not all of the APIs we build